changes to support ROSA 4.14.x and internal DB2 on ROSA..

NatarajBTI opened the pull request:
@shajeena @amitmangalvedkar - Please review this PR. It includes changes to support ROSA 4.14.x version and internal DB2 on ROSA.. Both these flows are verified. MAS + Manage is verified on Oracle.

shajeena reviewed (commented):
@NatarajBTI - op_versions is validated for 4.14.x , line 44 says supports 4.12.x . Will this work ?

NatarajBTI commented:
> @NatarajBTI - op_versions is validated for 4.14.x , line 44 says supports 4.12.x . Will this work ?

@shajeena - Addressed your review comment. Please check..thanks

shajeena reviewed (commented):
Looks good , Please align the if condiitons :)
